tests/product/tests/conftest.py

```diff
@@ -186,6 +186,11 @@ def pytest_generate_tests(metafunc):
     test cases registered in TEST_PARAMETERS and the values passed to
     relevant custom cmdline parameters such as --range.
     """
+
+    if "-k" in metafunc.config.invocation_params.args:
```

Comment:
can we use a full notation instead of `-k`?

Reply:
what about this comment?

Reply:
Comment is added